ledger-api/grpc-definitions/com/daml/ledger/api/v2/version_service.proto

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v2;
+
+import "com/daml/ledger/api/v1/version_service.proto";
+
+option java_outer_classname = "VersionServiceOuterClass";
+option java_package = "com.daml.ledger.api.v2";
+option csharp_namespace = "Com.Daml.Ledger.Api.V2";
+
+// Allows clients to retrieve information about the ledger API version
+// In V2 Ledger API this service is not available anymore. Use v2.VersionService instead.
```

Comment:
comment seems off, this is the v2.versionservice. I think you also deleted slightly more than you technically needed. E.g., the experimental static time feature will likely still exist although I suppose you can argue that noone gets it through the version endpoint.

Reply:
The former is fixed, thanks for that.
I don't understand the "you also deleted slightly more than you technically needed" part.

Reply:
I meant the fact that there are still experimental features that don't seem redundant in 3.0, i.e., they are not always on. Haven't looked deeply into it but at least static time seems to be one of them.

Reply:
I'd like to split out this topic, so we could improve on this for 3.0, as this seems low prio to me. Would you agree?